api/pyproject.toml

```diff
@@ -176,6 +176,7 @@ werkzeug = "~3.0.1"
 xinference-client = "0.9.4"
 yarl = "~1.9.4"
 zhipuai = "1.0.7"
+elasticsearch = "8.12.1"
```

Comment:
Is it using ElasticSearch v8 client to accessing v7 server in the test? Please avoid crossing versions for validation.